src/app/components/case/Workflow/components/StyledTable.tsx

```diff
@@ -9,9 +9,9 @@ export default styled(Table)`
     color: ${ themeColor("tint", "level4") };
   }
   td {
-    line-height: ${ themeSpacing(12) };
-    padding: ${ themeSpacing(1) } ${ themeSpacing(3) };
-
+    padding: ${ themeSpacing(3) };
+    line-height: 20px;
```

Comment:
Shouldn't this be `line-height: ${ themeSpacing(5) }`?

Reply:
Yes, will fix